pandas/core/index.py

```diff
@@ -1398,15 +1396,34 @@ def sym_diff(self, other, result_name=None):
         the_diff = sorted(set((self.difference(other)).union(other.difference(self))))
         return Index(the_diff, name=result_name)
 
-    def get_loc(self, key):
+    def get_loc(self, key, method=None):
         """
         Get integer location for requested label
 
+        Parameters
+        ----------
+        key : label
+        method : {None, 'pad'/'ffill', 'backfill'/'bfill', 'nearest'}
+            default: exact matches only.
+            pad / ffill: find the PREVIOUS index value if no exact match.
```

Comment:
if you want to get fancy, you can put the method docs in a module level variable (and use ``@Appender``), just a thought